src/main/java/org/opentripplanner/util/time/DurationUtils.java

```diff
@@ -4,6 +4,12 @@
 
 import java.time.Duration;
 import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import lombok.val;
```

Comment:
```suggestion
import java.util.List;
import java.util.stream.Collectors;
```

Reply:
I think you are correct, I have fixed several bugs - it should adjust down again.